plugin.yaml

```diff
@@ -2,9 +2,9 @@ plugins:
 
   openstack:
     executor: central_deployment_agent
-    source: https://github.com/cloudify-cosmo/cloudify-openstack-plugin/archive/3.1.0.zip
+    source: https://github.com/cloudify-cosmo/cloudify-openstack-plugin/archive/3.1.1.zip
     package_name: cloudify-openstack-plugin
-    package_version: '3.1.0'
+    package_version: '3.1.1'
```

Comment:
@0lvin Right now I'm working on 3.1.1 and I add some fixes and updates, so please do not merge or release anything

Reply:
will be 3.2.0